app/src/main/res/values/strings.xml

```diff
@@ -537,7 +537,7 @@ Upload your first media by tapping on the add button.</string>
   <string name="previous_button_tooltip_message">Click to reuse the title and description you entered in your previous picture and modify it to fit your current one</string>
   <string name="welcome_do_upload_content_description">Examples of good images to upload to Commons</string>
   <string name="welcome_dont_upload_content_description">Examples of images not to upload</string>
-  <string name="skip_image">SKIP THIS IMAGE</string>
+  <string name="skip_image">Skip this image</string>
```

Comment:
Did you capitalize them programatically?

Reply:
no but i  checked all through regular expression

Reply:
What do you mean by "checked all through regular expression" can you please explain?

Reply:
![2019-06-11_14-40](https://user-images.githubusercontent.com/46790654/59259236-024aa300-8c57-11e9-90e1-4eb770dd5e94.png)
sorry for the late reply i checked them through the method shown in the screenshot .